src/Product.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function Product ({image, label, product, price}) {
+    return (
+        <article>
+            <span> {label} </span>
+            <img src={image}/>
+                <p>
+                    {product}
+                </p>
+                <h4>
+                    {price}
+                </h4>

```

Comment:
Je had hier het euro teken en de ,- in het component kunnen zetten wat minder werk geeft bij het aanroepen van het component. Maar als je ook andere valuta en centen zou willen gebruiken maakt dat je component weer minder flexibel. Echter meestal wil je je prijs als een getal houden en niet als een string zodat je er op kan sorteren sorteren op string gaat afabetisch en dan is 40009 minder dan 41.